pages/deployment/[uuid].tsx

```diff
@@ -225,6 +225,24 @@ export default function Deployment() {
                 </Alert>
               </Box>
             )}
+            {deployment.ungoverned && (
+              <Box>
+                <Alert
+                  severity='info'
+                  sx={{
+                    width: 'fit-content',
+                    m: 'auto',
+                    backgroundColor: '#0288d1',
```

Comment:
Are you changing the colours here because the standard info colours don't work for us? I feel like we should be able to rely on our theme here. If we want to use another alert with the same style somewhere else we'll have to copy/paste these colours. If we use the theme then we don't have to do anything aside from providing the severity.